App/VSTS/QuickActions.tsx

```diff
@@ -0,0 +1,81 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+// import { Office } from 'Office';
+import { ItemHyperlink } from  './ItemHyperlink';
+import { FollowButton } from './FollowButton';
+import { ReplyAllButton } from './ReplyAllButton';
+import { CopyButton } from './CopyButton';
+import { FollowStateTypes } from '../statesEZ';
+import { connect } from 'react-redux';
+import { RestButton } from './RestButton';
+import * as ReactDOM from 'react-dom/server';
+
+interface IQuickActionProps {
+  VSTShtmlLink?: string;
+  followState?: FollowStateTypes;
+  id?: string;
+  title?: string;
+  workItemType?: string;
+}
+
+function mapStateToProps(state: any): IQuickActionProps {
+  return {
+    VSTShtmlLink: state.quickActionState.VSTShtmlLink,
+    followState: state.quickActionState.followState,
+    id: state.quickActionState.id,
+    title: state.quickActionState.title,
+    workItemType: state.quickActionState.workItemType,
+  };
+}
+
+@connect(mapStateToProps)
+export class QuickActions extends React.Component<IQuickActionProps, {isReady: boolean}> {
```

Comment:
Ah I get it. This is a workaround for Office.initialize for now. You will remove this when you integrate? If yes then you can leave it as it is.